packages/gestalt/src/Video.js

```diff
@@ -25,35 +25,35 @@ type Props = {|
   /**
    * Accessibility label for the fullscreen maximize button if controls are shown.
    */
-  accessibilityMaximizeLabel: string,
+  accessibilityMaximizeLabel?: string,
```

Comment:
I think all these props can be removed, as they are all going to be the same, there's no cardinality on the labels.